src/applications/mhv/medical-records/containers/AllergyDetails.jsx

```diff
@@ -194,6 +199,10 @@ const AllergyDetails = props => {
               Location
             </h2>
             <p data-dd-privacy="mask">{allergy.location}</p>
+            <h2 className="vads-u-font-size--base vads-u-font-family--sans">
+              Observed or historical
+            </h2>
+            <p data-dd-privacy="mask">{allergy.observedOrReported}</p>
```

Comment:
You will need to change the field value (in constants.js) to include the full text in the sketch view:
- Observed (you experienced this allergy or reaction while you were getting care at this VA location)
- Historical (you experienced this allergy or reaction in the past, before you started getting care at this VA location)